js/src/volume.js

```diff
@@ -246,6 +246,9 @@ var ScatterView = widgets.WidgetView.extend( {
                 animation_time_vz : { type: "f", value: 1. },
                 animation_time_size : { type: "f", value: 1. },
                 animation_time_color : { type: "f", value: 1. },
+                animation_time_index : { type: "f", value: 1. },
```

Comment:
Say if 'x' is a sequence, you'd set:
```javascript
this.previous_values["x"] = this.model.get('x')[this.model.previous('sequence_index')]
this.attributes_changed['x'] = ['x']
```
Now (via calling this.update, and then arriving at create_mesh)
It would (or should) detect that x has changed, and it will fill the `offsets_previous` with the this.previous_values["x"] values, and the `offsets` with the current `x[sequence_index]` values. I think that should work. Otherwise update this PR so I can see the new code.